include/aare/CalculateEta.hpp

```diff
@@ -0,0 +1,120 @@
+#pragma once
+
+#include "aare/Cluster.hpp"
+#include "aare/ClusterVector.hpp"
+#include "aare/NDArray.hpp"
+
+namespace aare {
+
+typedef enum {
+    cBottomLeft = 0,
+    cBottomRight = 1,
+    cTopLeft = 2,
+    cTopRight = 3
+} corner;
+
+typedef enum {
+    pBottomLeft = 0,
+    pBottom = 1,
+    pBottomRight = 2,
+    pLeft = 3,
+    pCenter = 4,
+    pRight = 5,
+    pTopLeft = 6,
+    pTop = 7,
+    pTopRight = 8
+} pixel;
+
+// TODO: maybe template this!!!!!! why int32_t????
+struct Eta2 {
```

Comment:
The clusters come 99% of the time from 16 bit (ok 14bit) integers. To make sure we can hold the sum of the cluster it needs to be int32_t. But yes if the code should work with other types of clusters we need to template this.